test/Makefile

```diff
@@ -3,6 +3,14 @@ ifdef OPENMP
   CFLAGS += -fopenmp
 endif
 
+TARGET := $(shell $(CC) -dumpmachine)
+ifneq (, $(findstring darwin, $(TARGET)))
+BENCH_LDFLAGS=
+else
+# default to linux, -lrt needed
+BENCH_LDFLAGS=-lrt
+endif 
```

Comment:
Will remove the trailing space after `endif` and add 2 spaces for indentation as is done for in the OPENMP block